go1.14/main/Dockerfile.tmpl

```
@@ -1,12 +1,12 @@
 ARG REPOSITORY
 ARG VERSION
-ARG TAG_EXTENSION=
+ARG TAG_EXTENSION
```

Similar to the other templates

```suggestion
ARG TAG_EXTENSION=''
```